behave_django/runner.py

```diff
@@ -0,0 +1,62 @@
+try:
+    from django.test.runner import DiscoverRunner
+except ImportError:
+    from django.test.simple import DjangoTestSuiteRunner as DiscoverRunner
+
+from django.core.management import call_command
+
+try:
+    from django.shortcuts import resolve_url
+except ImportError:
+    # support Django 1.4, which has no resolve_url() yet
+    from django.shortcuts import redirect
+
+    resolve_url = lambda to, *args, **kwargs: \
+        redirect(to, *args, **kwargs)['Location']
+
+from behave_django.testcase import (BehaviorDrivenTestCase,
+                                    ExistingDatabaseTestCase)
+
+
+class BehaviorDrivenTestRunner(DiscoverRunner):
+    testcase_class = BehaviorDrivenTestCase
+
+    def before_scenario(self, context):
```

Comment:
Why did you decide to put the code for the hooks here, rather than a separate module?